hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -109,13 +109,13 @@
   private String checksumType = ChecksumType.CRC32.name();
 
   @Config(key = "bytes.per.checksum",
-      defaultValue = "1MB",
+      defaultValue = "256KB",
```

Comment:
Let's not change the default for now. We can change once we do some tests and analyze performance .

Reply:
Sure. Reverted back to 1MB.